src/frontend/web_application/src/components/DropdownMenu/index.jsx

```diff
@@ -6,15 +6,24 @@ import './style.scss';
 
 export { withDropdownControl };
 
-const DropdownMenu = ({ className, ...props }) => (
-  <Dropdown className={classnames('m-dropdown-menu', className)} {...props} />
+const DropdownMenu = ({ className, hasTriangle, ...props }) => (
+  <Dropdown
+    className={classnames(
+      'm-dropdown-menu',
+      { 'm-dropdown-menu--has-triangle': hasTriangle },
```

Comment:
imo DropdownMenu has its very own style (blue bg...) and too keep future changes consistent we should use it here instead of customizing Dropdown.